docs/reference/indices/index-mgmt.asciidoc

```diff
@@ -51,7 +51,9 @@ You can also filter indices using the search bar.
 
 You can drill down into each index to investigate the index
 <<index-modules-settings,settings>>, <<mapping,mapping>>, and statistics.
-From this view, you can also edit the index settings.
+From this view, you can also edit the index settings. 
+
+To view and explore the documents within an index, click the compass symbol next to the index name to open {kibana-ref}/discover.html[Discover].
```

Comment:
We use "icon" instead of "symbol." You should also include an inline image of the icon.

Example from the [Maps docs](https://www.elastic.co/guide/en/kibana/current/maps-search.html):

<img width="869" alt="Screenshot 2023-07-24 at 7 30 39 AM" src="https://github.com/elastic/elasticsearch/assets/40268737/c8416076-6b8f-4f4f-aa7d-ca5794c6adbe">

[Source Asciidoc](https://github.com/elastic/kibana/blob/main/docs/maps/search.asciidoc)